pkg/secretless/plugin/connector/http/http.go

```diff
@@ -15,18 +15,28 @@ type Plugin interface {
 	NewConnector(connector.Resources) Connector
 }
 
-// Connector is an interface with a single method, "Connect".  "Connect" is a
-// function that knows how to authenticate an http request. It uses
-// its credential values argument passed in to modify the http.Request
-// request object and the credentials map to authenticate the client.
+/*
+Connector is an interface with a single method, Connect. Connect is a
+function that knows how to authenticate an http request. It uses
+its credential values argument to modify the http.Request and authenticate the
+client.
+
+Typically altering the HTTP request so that it contains the necessary
+authentication information means adding the appropriate headers to the request
+-- for example, an `Authorization` header containing a Token, or a header
```

Comment:
I don't think "Token" should be uppercase unless you are referring to a specific token by name. If a more generic "token" lowercase is fine.

Reply:
it's referring to the specific bit of metadata in the `Authorization` header, so I think `Token` is correct - but I removed the backticks around Authorization to be consistent.

Reply:
Sounds good!